services/blockchain-indexer/shared/dataService/business/interoperability/blockchainApps.js

```diff
@@ -44,11 +46,34 @@ const getBlockchainApps = async (params) => {
 
 	const total = await blockchainAppsDB.count(params);
 
-	blockchainAppsInfo.data = await blockchainAppsDB.find(
+	const response = await blockchainAppsDB.find(
 		{ ...params, limit: params.limit || total },
 		Object.getOwnPropertyNames(blockchainAppsIndexSchema.schema),
 	);
 
+	blockchainAppsInfo.data = await BluebirdPromise.map(
+		response,
+		async (appInfo) => {
+			if (!appInfo.isDefault) {
+				const isDefault = !!config.defaultApps.some(e => e.includes(appInfo.name));
```

Comment:
`appInfo.name` should be an exact match with one of the entries within the `config.defaultApps`.

```suggestion
				const isDefault = config.defaultApps.some(e => e === appInfo.name);
```